force-app/main/educationCloudSettings/lwc/educationCloudSettings/educationCloudSettings.html

```diff
@@ -1,9 +1,9 @@
 <template>
     <div class="slds-theme_default eda-height_full">
         <ul class="slds-grid slds-wrap">
-            <li class="slds-col slds-size_12-of-12">
+            <li class="slds-col slds-size_12-of-12 slds-var-m-bottom_large">
                 <h2 class="slds-text-heading_medium eda-text_heading_bold slds-var-m-left_small slds-var-m-top_small">
-                    Release Gating
+                    Release Gating (test)
```

Comment:
We probably need to create a label here. We can do this later, before the merge into `feature/234`